App.js

```diff
@@ -1,26 +1,326 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+import { View, Vibration } from 'react-native'
+import LottieView from 'lottie-react-native'
+
 
 const App = () => {
+  const [question, setQuestion] = useState('begin')
+  const [score, setScore] = useState(0)
+  const [hide, setHide] = useState(false)
+  const [right, setRight] = useState(false)
+  const [wrong, setWrong] = useState(false)
+  const duration = 600
+
+  const afterAnswering = () => {
+    setHide(false)
+    setRight(false)
+    setWrong(false)
+  }
+
+  const nextQuestion = () => {
+    if(question==='begin') {
+      setQuestion('one')
+    } else if(question==='one') {
+      setQuestion('two')
+      afterAnswering()
+    } else if(question==='two') {
+      setQuestion('three')
+      afterAnswering()
+    } else if(question==='three') {
+      setQuestion('four')
+      afterAnswering()
+    } else if(question==='four') {
+      setQuestion('five')
+      afterAnswering()
+    } else if(question==='five') {
+      setQuestion('summary')
+      afterAnswering()
+    }
+  }
+
+  const rightAnswer = () => {
+    setScore(score+1)
+    setHide(true)
+    setRight(true)
+  }
+
+  const wrongAnswer = () => {
+    setHide(true)
+    setWrong(true)
+    return Vibration.vibrate(duration)
+  }
+
+  const restartQuiz = () => {
+    setScore(0)
+    setQuestion('begin')
+  }
+
+
   return (
     <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
+      {question === 'begin' && (
+      <View>
+        <Container>
+        <Title>Welcome to Michels movie quiz</Title>
+        <Title>🍿🍿🍿</Title>
+        <Title>❓❓❓</Title>
+        <Button>
+          <ButtonText onPress={() => setQuestion('one')}>Start the quiz</ButtonText>
+        </Button>
+        <LottieView source={require('./1879-movie-loading.json')} autoPlay loop style={{height: 100}}/>
```

Comment:
Yay! You managed to add a Lottie animation. This is super useful knowledge if you're getting into more react native projects.